airflow/.docker/entrypoint.sh

```diff
@@ -59,12 +59,18 @@ wait_for_redis() {
   fi
 }
 
+install_csv_generator() {
+    # hack: will be removed once `csv_generator` is hosted on `pypi`
+    pip install --user csv-generator/dist/csv_generator-0.0.1-py3-none-any.whl
```

Comment:
I meant to ask where do we get that file from. The dev needs to run the setup?

Reply:
Yes, (just adding the readme entry).

I am just putting it in a script at present (as this step will be removed at some point) and will add in the `db_manager` wheel file generation step to it once that's ready.